lib/env-bundler.js

```diff
@@ -1199,7 +1199,14 @@ async function setupEnvironment (options) {
     resourceDetails: `${coreBaseApi}/api/fileBrowser/file?protectedFolder=false&currentPath={path}`,
   };
 
-  token = await getToken();
+  try {
+    token = await getToken();
+  } catch (error) {
+    if (error.isAxiosError && error.message.includes('certificate')) {
```

Comment:
This seems a weak check. Can you improve it someway?

Reply:
Unfortunately there is no generic "invalid certificate" error, since the error code is provided directly by OpenSSL, that is very specific: https://nodejs.org/api/tls.html#x509-certificate-error-codes